chef/data_bags/crowbar/migrate/provisioner/034_os_upgrading_state.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade(ta, td, a, d)
+  a["dhcp"]["state_machine"]["os_upgrading"] = ta["dhcp"]["state_machine"]["os_upgrading"]
+  a["dhcp"]["state_machine"]["os_upgraded"] = ta["dhcp"]["state_machine"]["os_upgraded"]
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
```

Comment:
Unused method argument - `ta`. If it's necessary, use `_` or `_ta` as an argument name to indicate that it won't be used.<br>Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.